packages/server/src/validator/utils.ts

```diff
@@ -12,9 +12,17 @@ export function validateParam(index: number | string, param: any, validation: an
 
   if (requiredIsMissing(param, validation.required)) {
     throw predefined.MISSING_REQUIRED_PARAMETER(index);
+  } else if (!validation.required && param === undefined) {
+    //if parameter is undefined and not required, no need to validate
+    //e.g estimateGas method, blockNumber is not required
+    return;
   }
```

Comment:
nit: this will be slightly easier to read:
```suggestion
  if (param === undefined) {
    if (validation.required) {
      throw predefined.MISSING_REQUIRED_PARAMETER(index);
    } else {
      // if parameter is undefined and not required, no need to validate
      // e.g estimateGas method, blockNumber is not required
      return;
    }
  }
```

Reply:
For some reason when I introduce the new change you suggested the unit tests start failing in the CI